args: ['python3', 'tulip.py', './tests/type_check_errors/dynamic_signature_bad_inputs.tlp']
-------- stdout ----------
--------------------------
-------- stderr ----------
./tests/type_check_errors/dynamic_signature_bad_inputs.tlp:1:7 [ERROR]: Invalid inputs for OpType.INTRINSIC:Intrinsic.SPLIT.
    [Note]: Found [DataType(Ident='int', Generic=False, Struct=False, Size=1)]
--------------------------
exit code: 1
